### PR TITLE
Add release notes for `v1.0.5`

### PR DESCRIPTION
**Pull request by Gedochao**
Relevant ticket: #2469

### CODE REVIEW

File: website/docs/release_notes.md [hunk 1]
```diff
@@ -7,9 +7,173 @@ import ReactPlayer from 'react-player'
...
+scala-cli compile . --offline --power
+```
+
+Of course, this means that you will have to have all the dependencies relevant to your build 
+already downloaded and available in your local cache.
+Reasonable fallbacks will be used where possible, 
+e.g. the Scala compiler may be used instead of Bloop if Bloop isn't available.
+
+Added by [@MaciejG604](https://github.com/MaciejG604) in [#2404](https://github.com/VirtusLab/scala-cli/pull/2404)
+
```
Comment (MaciejG604) [inline website/docs/release_notes.md:43]:
```suggestion
```

File: website/docs/release_notes.md [hunk 2]
```diff
@@ -310,7 +476,7 @@ a warning message when such non-compliant options are passed.
   in [#2207](https://github.com/VirtusLab/scala-cli/pull/2207)
 * Update guava to 32.0.1-jre by [@scala-steward](https://github.com/scala-steward-org/scala-steward)
   in [#2197](https://github.com/VirtusLab/scala-cli/pull/2197)
-* Update scala-cli.sh launcher for 1.0.1 by @github-actions in [#2194](https://github.com/VirtusLab/scala-cli/pull/2194)
+* Update scala-cli.sh launcher for 1.0.1 by [@github-actions](https://github.com/github-actions) in [#2194](https://github.com/VirtusLab/scala-cli/pull/2194)
```
Comment (MaciejG604) [inline website/docs/release_notes.md]: The link is incorrect, I think that's why there was none.

**Review by MaciejG604 — APPROVED** [review]
LGTM

**Review by tgodzik — APPROVED** [review]
Looks good, though we are blocked by the M1 docker :(

File: website/docs/release_notes.md (hunk 2 above)
Comment (Gedochao) [reply website/docs/release_notes.md]: y're right, my bad.
fixed it to https://github.com/features/actions

**Pull request merged**